pkgs/by-name/st/startup-disk/package.nix

```
@@ -0,0 +1,67 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+  nix-update-script,
+  pkg-config,
+  libadwaita,
+  gtk4,
+  glib,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "startup-disk";
+  version = "0.1.5";
+
+  src = fetchFromGitLab {
+    owner = "davide125";
+    repo = "startup-disk";
+    tag = "${version}";
+    hash = "sha256-258whEX6hKqfrk2aII15tuFEuB7NQUCNLEmi3OCOWV4=";
+    domain = "gitlab.gnome.org";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    glib.dev
+  ];
+
+  buildInputs = [
+    gtk4
+    libadwaita
+  ];
+
+  postPatch = ''
+    # Fix sudo crate's hardcoded /usr/bin/sudo
+    substituteInPlace ${"$"}{cargoDepsCopy}/sudo-0.6.0/src/lib.rs \
```

```suggestion
    substituteInPlace $cargoDepsCopy/sudo-0.6.0/src/lib.rs \
```

Yeah there's no need to complicate this